### PR TITLE
Calculation improvement

### PR DESCRIPTION
**Pull request by LeaveMyYard**
- Increased memory buffer
- Changed default step to 1m15s (1.25m)
- Fixed duration and step formatting in queries

### CODE REVIEW

**Comment by fcecagno** [comment]
Just tested this and found it odd that all my deployments had recommendation of 100m CPU request, even those that consumed almost no CPU in the past (for instance, `config-reloader`).

**Comment by LeaveMyYard** [comment]
@fcecagno I think this is because we set the minimal recommendation to 100m for CPU (previously it was 5m I think). @Avi-Robusta Don't you think that we should make it less?

File: robusta_krr/strategies/simple.py [hunk 1]
```diff
@@ -75,14 +94,22 @@ def __calculate_memory_proposal(
         self, history_data: MetricsPodData, object_data: K8sObjectData
     ) -> ResourceRecommendation:
         data = history_data["MaxMemoryLoader"]
+        data_count = {pod: values[0, 1] for pod, values in history_data["MemoryAmountLoader"].items()}
```
Comment (RoiGlinik) [inline robusta_krr/strategies/simple.py:97]: this can be done after checking if data is empty

File: robusta_krr/strategies/simple.py [hunk 2]
```diff
@@ -55,16 +66,24 @@ class SimpleStrategy(BaseStrategy[SimpleStrategySettings]):
...
     @property
     def metrics(self) -> list[type[PrometheusMetric]]:
-        return [PercentileCPULoader(self.settings.cpu_percentile), MaxMemoryLoader]
+        return [PercentileCPULoader(self.settings.cpu_percentile), MaxMemoryLoader, CPUAmountLoader, MemoryAmountLoader]
 
     def __calculate_cpu_proposal(
         self, history_data: MetricsPodData, object_data: K8sObjectData
     ) -> ResourceRecommendation:
         data = history_data["PercentileCPULoader"]
+        data_count = {pod: values[0, 1] for pod, values in history_data["CPUAmountLoader"].items()}
```
Comment (RoiGlinik) [inline robusta_krr/strategies/simple.py:75]: can be done after checking if data is empty

**Review by RoiGlinik — APPROVED** [review]

**Pull request merged**